packages/mongo-nav/src/mongo-select/OrgSelect.tsx

```diff
@@ -340,106 +336,115 @@ function OrgSelect({
+            data-testid="org-trigger"
+            disabled={loading}
+            onClick={onElementClick(
+              NavElement.OrgNavOrgSelectTrigger,
+              toggleOpen,
+            )}
+            className={cx(baseButtonStyle, orgButtonStyle, {
+              [activeButtonStyle]: open,
               [textLoadingStyle]: loading,
+              [orgStandaloneButtonStyle]: disabled,
             })}
           >
-            {disabled ? 'All Organizations' : current?.orgName ?? ''}
-          </span>
-          {open ? (
-            <CaretUpIcon size="small" className={caretIconStyle} />
-          ) : (
-            <CaretDownIcon size="small" className={caretIconStyle} />
-          )}
-        </button>
-        <Menu
-          usePortal={false}
-          className={menuContainerStyle}
-          justify="start"
-          spacing={0}
-          setOpen={toggleOpen}
-          open={open}
-        >
-          {data && (
-            <FocusableMenuItem>
-              <Input
-                data-testid="org-filter-input"
-                variant="organization"
-                onChange={onChange}
-                onKeyDown={onKeyDown}
-                value={value}
+            {!isTablet && (
+              <BuildingIcon
+                size="small"
+                className={cx(iconColorStyle, { [iconLoadingStyle]: loading })}
               />
-            </FocusableMenuItem>
-          )}
-
-          <ul className={ulStyle}>
-            {isAdminSearch && isFetching && (
-              <li className={emptyStateStyle}>Searching...</li>
             )}
-            {isAdminSearch && !isFetching && renderedData.length === 0 && (
-              <li className={emptyStateStyle}>No matches found</li>
+            <span
+              data-testid="org-select-active-org"
+              className={cx(selectedStyle, {
+                [textLoadingStyle]: loading,
+              })}
+            >
+              {disabled ? 'All Organizations' : current?.orgName ?? ''}
+            </span>
+            {open ? (
+              <CaretUpIcon size="small" className={caretIconStyle} />
+            ) : (
+              <CaretDownIcon size="small" className={caretIconStyle} />
             )}
-            {renderedData?.map(renderOrganizationOption) ?? (
-              <li className={emptyStateStyle}>
-                You do not belong to any organizations. Create an organization
-                on the{' '}
-                <a href={urls.viewAllOrganizations} className={linkStyle}>
-                  Organizations
-                </a>{' '}
-                page.
-              </li>
+          </button>
+          <Menu
+            usePortal={false}
+            className={menuContainerStyle}
+            justify="start"
+            spacing={0}
+            setOpen={toggleOpen}
+            open={open}
+          >
+            {data && (
+              <FocusableMenuItem>
+                <Input
```

Comment:
Input element not rendering properly anymore 

<img width="448" alt="Screen Shot 2020-11-17 at 11 26 45 AM" src="https://user-images.githubusercontent.com/26016393/99417503-fe996200-28c7-11eb-9f02-bde8557d1be3.png">

Reply:
seems to be an existing bug too but I have a fix